test/fixtures/wpt/url/resources/urltestdata.json

```diff
@@ -5317,17 +5317,12 @@
   {
     "input": "http://256.256.256.256.256",
     "base": "http://other.com/",
-    "href": "http://256.256.256.256.256/",
-    "origin": "http://256.256.256.256.256",
-    "protocol": "http:",
-    "username": "",
-    "password": "",
-    "host": "256.256.256.256.256",
-    "hostname": "256.256.256.256.256",
-    "port": "",
-    "pathname": "/",
-    "search": "",
-    "hash": ""
+    "failure": true
+  },
+  {
+    "input": "http://256.256.256.256.256.",
+    "base": "http://other.com/",
+    "failure": true
```

Comment:
`test/fixtures/wpt` should be updated using https://github.com/nodejs/node-core-utils/blob/main/docs/git-node.md#git-node-wpt

Reply:
This will introduce more failing tests. I've opened an additional issue in order to keep track of spec compliance and fix those issues.